python/xorbits/_mars/dataframe/base/__init__.py

```diff
@@ -86,7 +85,6 @@ def _install():
         setattr(
             t, "__delitem__", lambda df, items: df_drop(df, items, axis=1, inplace=True)
         )
-        setattr(t, "dedup", df_dedup)
```

Comment:
The `dedup()` method is unregistered from mars DataFrame. How can I use the dedup feature?

Reply:
```python
from xorbits.experimental import dedup

df = # your data

res = dedup(df, col="text")
```

Reply:
dedup is not the feature of dataframe anymore, it belongs to a new collection named experimental

Reply:
It's our extension to pandas DataFrame. That makes sense.